src/subscription_manager/identity.py

```diff
@@ -169,3 +169,6 @@ def __str__(self):
         return "<%s, name=%s, uuid=%s, consumer=%s>" % \
                 (self.__class__.__name__,
                 self.name, self.uuid, self.consumer)
+
+    def log_str(self):
```

Comment:
Looks like nothing uses Identity.__str__, so I'll change it.

(or at least, raising an exception in Identity.__str__ didn't break any unittests...)